docs/languages/powershell.md

```diff
@@ -0,0 +1,128 @@
+# Editing PowerShell in Visual Studio Code
+
+PowerShell is a task-based command line shell and scripting language built on the .NET framework.
+Powershell provides a powerfull toolset for administrators on any platform.
+
+The PowerShell extension provides rich [PowerShell Language Support](https://github.com/PowerShell/vscode-powershell)
+
+## Install the PowerShell extension on Visual Studio Code
+
+### Installing the Extension
```

Comment:
It looks like there's no other subheading for this, so maybe we can just remove it and subsume the content under the H2?